app/models/content_item.rb

```diff
@@ -2,7 +2,7 @@ class ContentItem
   include Mongoid::Document
   include Mongoid::Timestamps::Updated
 
-  NON_RENDERABLE_FORMATS = %w(redirect gone)
+  NON_RENDERABLE_FORMATS = %w(redirect gone).freeze
```

Comment:
I've updated this PR not to include this. Will fire in another with that constant removed.